source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACOBaseActionElement.h

```diff
@@ -28,6 +28,7 @@
 - (NSData *)additionalProperty;
 - (BOOL)isEnabled;
 - (BOOL)meetsRequirements:(ACOFeatureRegistration *)featureReg;
+- (BOOL)shouldFlipInRtl;
```

Comment:
nit: if you make it a readonly computed property the consumer can use the `.` syntax to access this flag.

Reply:
The `.m` implementation will remain same.